internal/pkg/cli/deploy/deploy.go

```diff
@@ -414,7 +414,13 @@ type StackRuntimeConfiguration struct {
 // DeployWorkloadInput is the input of DeployWorkload.
 type DeployWorkloadInput struct {
 	StackRuntimeConfiguration
-	ForceNewUpdate bool
+	DeployOptions
+}
+
+// DeployOptions specifies options for the deployment.
+type DeployOptions struct {
```

Comment:
nit: What do you think of just
```suggestion
// Options holds not-required fields to deploy a stack.
type Options struct {
```
so that it reads `deploy.Options`